cmd/clusterctl/client/repository/metadata_client.go

```diff
@@ -156,6 +156,20 @@ func (f *metadataClient) getEmbeddedMetadata() *clusterctlv1.Metadata {
 					// older version are not supported by clusterctl
 				},
 			}
+		case config.EKSBootstrapProviderName:
+			return &clusterctlv1.Metadata{
+				TypeMeta: metav1.TypeMeta{
+					APIVersion: clusterctlv1.GroupVersion.String(),
+					Kind:       "Metadata",
+				},
+				ReleaseSeries: []clusterctlv1.ReleaseSeries{
+					// v1alpha3 release series
+					{Major: 0, Minor: 2, Contract: "v1alpha3"},
```

Comment:
Am I wrong or the EKS provider version is tight to the CAPA provider version? 
```suggestion
					{Major: 0, Minor: 6, Contract: "v1alpha3"},
```